enterprise/admin/views.py

```diff
@@ -102,6 +107,32 @@ def get_user_name(request):
         return request.user.first_name or request.user.username
 
 
+class CatalogQueryPreviewView(APIView):
```

Comment:
Should we add admin-only access validation on this view.

Reply:
These are only for the admin site